_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -151,19 +151,44 @@ function is_wp_version_too_old() {
 	 * @return bool|array
 	 */
 	function check_plan_deactivate_modules( $page ) {
-		if ( Jetpack::is_development_mode()
-			|| ! in_array( $page->base, array( 'toplevel_page_jetpack', 'admin_page_jetpack_modules', 'jetpack_page_vaultpress', 'jetpack_page_stats', 'jetpack_page_akismet-key-config' ) )
-			|| true === self::$plan_checked ) {
+		if (
+			Jetpack::is_development_mode()
+			|| ! in_array(
+				$page->base,
+				array(
+					'toplevel_page_jetpack',
+					'admin_page_jetpack_modules',
+					'jetpack_page_vaultpress',
+					'jetpack_page_stats',
+					'jetpack_page_akismet-key-config'
+				)
+			)
+			|| true === self::$plan_checked
+		) {
 			return false;
 		}
+
 		self::$plan_checked = true;
 		$previous = get_option( 'jetpack_active_plan', '' );
 		$response = rest_do_request( new WP_REST_Request( 'GET', '/jetpack/v4/site' ) );
+
+		if ( $response->is_error() ) {
```

Comment:
for extra safety, how about:

`if ( ! is_object( $response ) || $response->is_error() ) {`